rust/lancedb/src/remote/table.rs

```diff
@@ -999,6 +999,18 @@ impl<S: HttpSend> BaseTable for RemoteTable<S> {
             "column": column
         });
 
+        // Add name parameter if provided (for backwards compatibility, only include if Some)
+        if let Some(ref name) = index.name {
+            body["name"] = serde_json::Value::String(name.clone());
+        }
+
+        // Warn if train=false is specified since it's not supported in remote tables
+        if !index.train {
+            log::warn!(
+                "train=false is not supported for remote tables. The index will be created empty and automatically updated in the background."
+            );
```

Comment:
```suggestion
        // Warn if train=false is specified since it's not meaningful
        if !index.train {
            log::warn!(
                "train=false has no effect remote tables. The index will be created empty and automatically populated in the background."
            );
```